docs/content/configuring-npm/package-json.md

```diff
@@ -823,16 +823,16 @@ module will run on:
 "os" : [ "darwin", "linux" ]
 ```
 
-You can also blacklist instead of whitelist operating systems,
-just prepend the blacklisted os with a '!':
+You can also denylist instead of allowlist operating systems,
+just prepend the denylisted os with a '!':
```

Comment:
I think this would need to be rephrased a little bit to make more sense, maybe:

```suggestion
You can also block instead of allowing operating systems,
just prepend the blocked os with a '!':
```